source/community.html.haml

```diff
@@ -28,14 +28,14 @@ title: Community
       %tr
         %td Users channel
         %td(colspan="2")
-          = link_to '#mpv on irc.freenode.net', 'irc://irc.freenode.net/mpv'
+          = link_to '#mpv on irc.libera.chat', 'ircs://irc.libera.chat:6697/mpv'
```

Comment:
```suggestion
          = link_to '#mpv on irc.libera.chat', 'ircs://irc.libera.chat/mpv'
```
I sure hope that this'd work automatically

Reply:
It doesn't, I tried.